server/app/assets/stylesheets/styles.css

```diff
@@ -14,6 +14,10 @@
     @apply text-xl;
   }
 
+  body {
+    color: rgb(0 0 0);
+  }
+
```

Comment:
suggestion (blocking): please add a comment clarifying the need for this rule

Reply:
Great callout.  I realize now that I need to add more comments in general.  Will add one here.

Reply:
Thanks!